AUTHORS.rst

```diff
@@ -0,0 +1,32 @@
+IMAPClient was created and is maintained by Menno Smits <menno@freshfoo.com>.
+Thanks go to the following people for their help with this project.
+
+Contributors
+~~~~~~~~~~~~
+| Mathieu Agopian
+| Chris Arndt
+| Jp Calderone
+| John Louis del Rosario
+| Dave Eckhardt
+| Eben Freeman
+| Helder Guerreiro
+| Mark Hammond <mhammond _AT_ skipinet.com.au>
+| Johannes Heckel
+| Thomas Jost <schnouki _AT_ schnouki.net>
+| Maxime Lorant
+| Lukasz Mierzwa
+| Naveen Nathan
+| Brian Neal
+| Phil Peterson
+| Aviv Salem
+| Andrew Scheller
+| Thomas Steinacher
+| Zac Witte
+
+Other Contributions
+~~~~~~~~~~~~~~~~~~~
+Nylas supports development of IMAPClient.
+http://nylas.com/
+
+The project was started while the author was working at NetBox Blue.
+http://netboxblue.com/
```

Comment:
This section can go. Nylas no longer supports the project and NetBox Blue was a *very* long time ago.